packages/alert-dialog/README.md

```diff
@@ -164,17 +208,21 @@ Destructive alert dialogs are for when a user needs to confirm an action that wi
 </sp-alert-dialog>
 ```
 
-### Secondary Button
+</sp-tab-panel>
+</sp-tabs>
+
+### Behaviors
```

Comment:
Lets add a behavior section that should the alert dialog in an overlay with the correct accessibility set up. we can make a follow up ticket to add it to storybook and clean up the button spacing issue